Gofer.NET.Tests/GivenATaskInfo.cs

```diff
@@ -80,5 +89,11 @@ private void TestMethod2()
         {
             Console.WriteLine(nameof(TestMethod2));
         }
+
+        private Task<string> TestMethod3(CancellationToken cancellation = default)
```

Comment:
No need for default parameter here, and may cause improper usage in future PRs.

Reply:
Done:)